Gems/AtomLyIntegration/AtomViewportDisplayIcons/Code/Source/AtomViewportDisplayIconsSystemComponent.h

```diff
@@ -79,6 +82,26 @@ namespace AZ
             IconId m_currentId = 0;
 
             bool m_drawContextRegistered = false;
+
+            AZStd::unordered_map<IconId, AZStd::vector<DrawParameters>> m_drawRequests;
```

Comment:
missing includes
#include <AzCore/std/containers/unordered_map.h>
#include <AzCore/std/containers/vector.h>
and atom clases like Instance etc

Reply:
I think I can make this a lot better using a small impl pattern actually.  I hate including headers in headers.

Reply:
I took a swing at it then realised that the impl pattern here is silly to do since this is a component header and is thus not necessary to actually include in any other compilation unit, save for possibly the module header (its not API, nobody should include it).  

So I just added the headers we needed, I'll upload a new code drop